arborist/server.go

```diff
@@ -675,6 +682,37 @@ func (server *Server) handlePolicyOverwrite(w http.ResponseWriter, r *http.Reque
 	_ = jsonResponseFrom(updated, 201).write(w, r)
 }
 
+func (server *Server) handleBulkPoliciesOverwrite(w http.ResponseWriter, r *http.Request, body []byte) {
+	var policies []Policy
+	err := json.Unmarshal(body, &policies)
+	if err != nil {
+		msg := fmt.Sprintf("could not parse policy from JSON: %s", err.Error())
+		server.logger.Info("tried to create policy but input was invalid: %s", msg)
+		response := newErrorResponse(msg, 400, nil)
+		_ = response.write(w, r)
+		return
+	}
+
+	for _, policy := range policies {
+		if mux.Vars(r)["policyID"] != "" {
```

Comment:
maybe we could move what's in the `for` loop to its own function instead of duplicating the code in `handlePolicyOverwrite` and `handleBulkPoliciesOverwrite`?